Covinator/Covinator/Pages/PfizerVaccination.cshtml

```diff
@@ -1,116 +1,45 @@
 ﻿@page
 @model Covinator.Pages.PfizerVaccination
 @{
-    ViewData["Title"] = "Home page";
+    ViewData["Title"] = "Pfizer Vaccine";
     var pfizerVaccineDistributionAllocationss = (QuickType.PfizerVaccineDistributionAllocations[])ViewData["PfizerVaccineDistributionAllocations"];
     var currentWeek = ViewData["CurrentWeek"];
 }
-<style>
```

Comment:
Removed inline styling and moved to site.css so all the tables could have a uniform styling

Reply:
Definitely... good idea.